docs/Point SDK/Flutter/Events List.md

```diff
@@ -19,6 +19,7 @@ Tempo Events
 
 | **Name**                        | **Description**                                                                             | **Event payload**                                                                                                                                       |
 |---------------------------------|---------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `tempoTrackingDidUpdate`  | ETA update for every Tempo event sent to backend     | Object with the following properties:<ul><li>`eta`: Estimated time of arrival to a particular store</li><li>`etaDirection`: lessThan/greaterThan from the `eta` value</li><li>`destination`: Destination details of the store like address, location, and `destinationId`</li><li>`triggerChainId`: unique Id per Tempo session</li></ul> |
```

Comment:
Also the Event payload needs to be updated for GeoTrigger Events table above:
<img width="1100" alt="Screenshot 2024-08-27 at 1 27 20 pm" src="https://github.com/user-attachments/assets/d8017bb6-111b-4810-8672-fb751de31946">

Reply:
Updated! Thank you